src/main/java/hudson/tasks/test/MetaTabulatedResult.java

```diff
@@ -41,4 +41,9 @@
      */
     public abstract Collection<? extends TestResult> getFailedTests();
 
+    /**
+     * All skipped tests.
+     */
+    public abstract Collection<? extends TestResult> getSkippedTests();
```

Comment:
Careful, this is going to blow up for [implementations](https://github.com/search?q=org%3Ajenkinsci+%22extends+MetaTabulatedResult%22&type=Code) that predate this change as they will not implement this method. Must be non-abstract.